site_root/input/resources/profiles/StructureDefinition-kfdrc-sequencing-experiment.json

```diff
@@ -0,0 +1,37 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "kfdrc-sequencing-experiment",
+  "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-sequencing-experiment",
+  "version": "0.1.0",
+  "name": "kfdrc-sequencing-experiment",
+  "title": "Kids First DRC Sequencing Experiment",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Task",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Task",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Task",
+        "path": "Task"
+      },
+      {
+        "id": "Task.extension",
+        "path": "Task.extension",
+        "slicing": {
+          "discriminator": [
+            {
+              "type": "value",
+              "path": "url"
+            }
+          ],
+          "ordered": false,
+          "rules": "open"
+        }
+      }
+    ]
+  }
+}
```

Comment:
As discussed above, we may hold off creating a new profile without any additions/modifications.